mne/beamformer/_lcmv.py

```diff
@@ -161,7 +171,15 @@ def make_lcmv(info, forward, data_cov, reg=0.05, noise_cov=None, label=None,
     .. [2] Sekihara & Nagarajan. Adaptive spatial filters for electromagnetic
            brain imaging (2008) Springer Science & Business Media
     """
-    picks = _setup_picks(picks, info, forward, noise_cov)
+    if picks is not None:
+        warn('Specifying picks is deprecated and will be removed in 0.16. '
+             'Specifying the selection of channels in info and setting picks '
+             'to None will remove this warning.',
+             DeprecationWarning)
+
+        info = pick_info(info, picks)
```

Comment:
Should this be wrapped in a function? It's used a lot. (Or should it be obvious in the code since it's a deprecation warning that will be removed anyway?)